frontend/src/components/Card.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, ViewPropTypes } from 'react-native';
+
+const cardStyle = {
+  padding: 10,
+  opacity: 0.8,
```

Comment:
```suggestion
  opacity: .8,
```

we usually don't write the 0 there